testmon/pytest_testmon.py

```diff
@@ -316,6 +324,8 @@ def pytest_collection_modifyitems(self, session, config, items):
     @pytest.hookimpl(hookwrapper=True)
     def pytest_runtestloop(self, session):
         yield
+        if session.config.option.collectonly:
```

Comment:
Carson, how does it work with yield above?
sorry for my ignorance

Reply:
The `pytest_runtestloop` is a [pluggy](https://pluggy.readthedocs.io/en/latest/) hook, so any plugin can register for it, registering it with the `hookwrapper` option means you hook the start and end of that hook with yield giving control to all the regular hook registrants. In this case, it isn't needed since we are just going at the end and could have just asked to be the last plugin..not sure why the original author did that, but it seems a little risky to change at this point

Reply:
gotcha. thank you!